common/alerts.js

```diff
@@ -31,10 +32,34 @@
             return alerts.splice(index, 1);
         }
 
+        /**
+         * Will create a specific type of alert for url limitation
+         * @param {string} message
+         */
+        function addURLLimitAlert(message) {
```

Comment:
Is this a wrapper so that you can check if the alert already exists, and if it does don't add a new one?

Reply:
yeah. At first I was attaching the alert object to the root and checking it that way. This looks better and cleaner I think.